Pod/Classes/WPMediaPickerViewController.h

```diff
@@ -222,6 +222,9 @@
  *  @return An empty view to display or `nil` to not display any.
  *
  *  If this method is not implemented, a default UILabel will be displayed.
+ *
+ *  In case of (emptyViewControllerForMediaPickerController) and (emptyViewForMediaPickerController) are both implemented by a parent ViewController,
+ *  the first one has precedence over the second one to be presented to the user.
```

Comment:
What about a simpler statement like this one.
```suggestion
 *  `emptyViewControllerForMediaPickerController` takes precedence over this method.
```

Reply:
Agreed 😄!